### PR TITLE
Feature/ia 5/rotation exercise

### PR DESCRIPTION
**Pull request by MiguelAskar**
Rotation challenge is already finished, result is shown below:

### CODE REVIEW

File: ConstraintsMiguelAskar/ConstraintsMiguelAskar/Assets.xcassets/Contents.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (cmorales-globant) [inline ConstraintsMiguelAskar/ConstraintsMiguelAskar/Assets.xcassets/Contents.json]: This does not belong to this PR.

**Review by cmorales-globant — CHANGES_REQUESTED** [review]
Views are not covering the whole screen -Ignoring safe area-, plus on landscape, red view is larger than it should. 
Screen Shot 2021-08-01 at 12 32 45 PM

File: ConstraintsMiguelAskar/ConstraintsMiguelAskar/Assets.xcassets/Contents.json (hunk 1 above)
Comment (cmorales-globant) [reply ConstraintsMiguelAskar/ConstraintsMiguelAskar/Assets.xcassets/Contents.json]: Double checked and this was happening because the PR was targeting `main` whereas it should  target `feature/IA-5/UI-constraint-exercises` which is your base branch for Constraint challenges, updated it and looks fine now.

**Review by pablocalcagnino — CHANGES_REQUESTED** [review]
AS Camilo mentioned fix and commit again please

**Comment by MiguelAskar** [comment]
> Views are not covering the whole screen -Ignoring safe area-, plus on landscape, red view is larger than it should.
> Screen Shot 2021-08-01 at 12 32 45 PM

Fixed

**Comment by MiguelAskar** [comment]
> AS Camilo mentioned fix and commit again please

fixed:

**Review by pablocalcagnino — APPROVED** [review]

**Review by cmorales-globant — APPROVED** [review]
